static/app/views/starfish/views/appStartup/screenSummary/widgets.tsx

```diff
@@ -1,11 +1,128 @@
+import {useReleaseSelection} from 'sentry/views/starfish/queries/useReleases';
+import {STARFISH_CHART_INTERVAL_FIDELITY} from 'sentry/views/starfish/utils/constants';
+import {appendReleaseFilters} from 'sentry/views/starfish/utils/releaseComparison';
 import CountWidget from 'sentry/views/starfish/views/appStartup/screenSummary/countWidget';
+import DeviceClassBreakdownBarChart from 'sentry/views/starfish/views/appStartup/screenSummary/deviceClassBreakdownBarChart';
+import {YAxis, YAXIS_COLUMNS} from 'sentry/views/starfish/views/screens';
+import {useTableQuery} from 'sentry/views/starfish/views/screens/screensTable';
 
 import AppStartBreakdownWidget from './appStartBreakdownWidget';
 
+const YAXES = [YAxis.COLD_START, YAxis.WARM_START];
+
 function SummaryWidgets({additionalFilters}) {
+  const pageFilter = usePageFilters();
+  const location = useLocation();
+  const {query: locationQuery} = location;
+  const {
+    primaryRelease,
+    secondaryRelease,
+    isLoading: isReleasesLoading,
+  } = useReleaseSelection();
+
+  const query = new MutableSearch([...(additionalFilters ?? [])]);
+
+  const searchQuery = decodeScalar(locationQuery.query, '');
+  if (searchQuery) {
+    query.addStringFilter(prepareQueryForLandingPage(searchQuery, false));
+  }
+
+  const queryString = appendReleaseFilters(query, primaryRelease, secondaryRelease);
+
+  // Use single query for cold and warm starts by device classification and release
+  const {
+    data: startupDataByDeviceClass,
+    isLoading,
+    isError,
+  } = useTableQuery({
+    eventView: EventView.fromNewQueryWithPageFilters(
+      {
+        name: '',
+        fields: [
+          'avg(measurements.app_start_cold)',
+          'avg(measurements.app_start_warm)',
+          'device.class',
+          'release',
+        ],
+        yAxis: YAXES.map(val => YAXIS_COLUMNS[val]),
+        topEvents: '2',
+        query: queryString,
+        dataset: DiscoverDatasets.METRICS,
+        version: 2,
+        interval: getInterval(
+          pageFilter.selection.datetime,
+          STARFISH_CHART_INTERVAL_FIDELITY
+        ),
+      },
+      pageFilter.selection
+    ),
+    enabled: !isReleasesLoading,
+    referrer: 'api.starfish.mobile-startup-bar-chart',
+    initialData: {data: []},
+  });
+
+  const transformedData: {
+    [yAxisName: string]: {
+      [releaseVersion: string]: Series;
+    };
+  } = {};
+
+  YAXES.forEach(val => {
+    transformedData[YAXIS_COLUMNS[val]] = {};
+    if (primaryRelease) {
+      transformedData[YAXIS_COLUMNS[val]][primaryRelease] = {
+        seriesName: primaryRelease,
+        data: Array(['high', 'medium', 'low', 'Unknown'].length).fill(0),
+      };
+    }
+    if (secondaryRelease) {
+      transformedData[YAXIS_COLUMNS[val]][secondaryRelease] = {
+        seriesName: secondaryRelease,
+        data: Array(['high', 'medium', 'low', 'Unknown'].length).fill(0),
+      };
+    }
+  });
```

Comment:
nit: we do this in a few places now, should we move it to a util?

Reply:
Good point, I turned this into a separate util than the helper I made for the last time I copied code like this over.

I noticed it's generally the same so there's an opportunity to refactor and make them both work for the same purpose, but for now I'm going to keep them separate. There are a few things to generalize and I don't want to do too much in this PR that may affect the screens module.